README.md

```diff
@@ -66,4 +64,4 @@ Give a ⭐️ if you like this project!
 
 ## 📝 License
 
-This project is [MIT](./MIT.md) licensed. -->
\ No newline at end of file
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] As mentioned by the previous code reviewer. It's important to follow correctly the template provided by Microverse. This includes attaching the correct license to the `License` section of the README. [You can find it here.](https://github.com/microverseinc/readme-template/blob/master/MIT.md)